src/views/Give/ManageDonationModal.tsx

```diff
@@ -434,7 +436,9 @@ export function ManageDonationModal({
         <Grid item xs={4}>
           <Box>
             <Typography variant="h5" align="center">
-              {project ? ((parseFloat(totalDonated) / project.depositGoal) * 100).toFixed(1) + "%" : "N/A"}
+              {project
+                ? new BigNumber(totalDebt).div(project.depositGoal).multipliedBy(100).toFixed(DECIMAL_PLACES) + "%"
```

Comment:
with high precision math we should do multiplication first and then division right? Not sure if it makes a difference given how BigNumber works internally but think that's usually best practice

Reply:
Done.